pkg/helm/cluster.go

```diff
@@ -87,3 +89,32 @@ func (h *Helm) OverrideDesiredVersion(rls *output.ReleaseOutput) {
 		}
 	}
 }
+
+// filterIgnoredReleases is a helper function that removes charts that match a release name or chart name
+// provided by the user at runtime from the list of found charts in the cluster
+func filterIgnoredReleases(deployed []*release.Release, releaseIgnoreList []string, chartIgnoreList []string) []*release.Release {
+	// Filter out any ignored releases
+	filteredDeployed := make([]*release.Release, len(deployed))
+	filteredCount := 0
+
+	for _, release := range deployed {
+		isIgnoredRelease := false
+		isIgnoredChart := false
+		for _, ignoreListedRelease := range releaseIgnoreList {
+			if release.Name == ignoreListedRelease {
+				isIgnoredRelease = true
+			}
+		}
+		for _, ignoreListedChart := range chartIgnoreList {
+			if release.Chart.Name() == ignoreListedChart {
```

Comment:
In trying to add a test for ignoreChart, I hit a nil pointer exception on this. While we shouldn't have a release that doesn't have a chart, it probably makes sense to check for nil here before calling .Name()